app/src/components/ChatBot/chat-popover.tsx

```diff
@@ -49,7 +74,7 @@ export default function ChatPopover({
       >
         <PopoverTrigger asChild>
           <Button
-            className="fixed z-30 bottom-16 right-16"
+            className={`fixed z-30 ${getDynamicBottomPosition()} right-16 transition-all duration-300`}
```

Comment:
Let's make sure this works in a production build (using `npm run build`) as well, since sometimes Tailwind has issues with dynamically created class strings (as it extracts the used classes at compile time).